src/model/oncocore/FluxTumorMarker.js

```diff
@@ -60,14 +60,14 @@ class FluxTumorMarker extends FluxEntry {
 
 
     get receptorTypeCodeableConcept(){
-        if (!this._tumorMarker.findingTopicCode || !this._tumorMarker.findingTopicCode.value) return null;
+        if (!(this._tumorMarker.findingTopicCode && this._tumorMarker.findingTopicCode.value && this._tumorMarker.findingTopicCode.value.coding && this._tumorMarker.findingTopicCode.value.coding.length > 0)) return null;
         return this._tumorMarker.findingTopicCode.value.coding[0];
     }
     
     // Returns a human-readable display text string
     get receptorType() { 
-        if (!this._tumorMarker.findingTopicCode || !this._tumorMarker.findingTopicCode.value) return null;
-        return this._tumorMarker.findingTopicCode.value.coding[0].displayText.value;
+        if (!this.receptorTypeCodeableConcept) return null;
```

Comment:
These changes here are going to be overwritten shortly by any changes in the `mcode_09` branch. Would it be possible to use the 09 branch for your testing?

Reply:
If the mcode_09 branch is going to be merged into master soon then this PR can wait until that time.